### PR TITLE
LPS-78361 Apply new Management Bar

### PR DESCRIPTION
**Pull request by pedroqueiroz**

### CODE REVIEW

**Comment by pedroqueiroz** [comment]
ci:test:relevant

**Comment by pedroqueiroz** [comment]
ci:stop

**Comment by pedroqueiroz** [comment]
ci:test:relevant

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java [hunk 1]
```diff
@@ -110,6 +116,56 @@ public DDMDataProviderInstance fetchDataProviderInstance()
 		return _ddmDataProviderInstance;
 	}
 
+	public DropdownItemList getActionItemsDropdownItemList() {
+		return new DropdownItemList(
+			PortalUtil.getHttpServletRequest(_renderRequest)) {
```
Comment (pedroqueiroz) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java]: use _formAdminRequestHelper.getRequest() instead

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java [hunk 2]
```diff
@@ -186,8 +238,29 @@ public String getDisplayStyle() {
...
+			_renderRequest);
+
+		return new DropdownItemList(request) {
+			{
+				addGroup(
+					dropdownGroupItem -> {
+						dropdownGroupItem.setDropdownItems(
+							getFilterNavigationDropdownItems());
+						dropdownGroupItem.setLabel(
+							LanguageUtil.get(request, "filter-by-navigation"));
```
Comment (pedroqueiroz) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java]: use _formAdminRequestHelper.getRequest() instead

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java [hunk 3]
```diff
@@ -282,11 +396,35 @@ public String getUserPortraitURL(long userId) throws PortalException {
...
-		return DDMFormPermission.contains(
-			_ddmDataProviderRequestHelper.getPermissionChecker(),
-			_ddmDataProviderRequestHelper.getScopeGroupId(),
-			DDMActionKeys.ADD_DATA_PROVIDER_INSTANCE);
+	public ViewTypeItemList getViewTypesItemList() {
+		HttpServletRequest request = PortalUtil.getHttpServletRequest(
+			_renderRequest);
+
+		return new ViewTypeItemList(
+			request, getPortletURL(), getDisplayStyle()) {
```
Comment (pedroqueiroz) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java]: use _formAdminRequestHelper.getRequest() instead

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java [hunk 4]
```diff
@@ -345,32 +471,30 @@ protected DDMFormRenderingContext createDDMFormRenderingContext() {
...
-		}
+		ThemeDisplay themeDisplay = (ThemeDisplay)request.getAttribute(
+			WebKeys.THEME_DISPLAY);
 
-		OrderByComparator<DDMDataProviderInstance> orderByComparator = null;
+		return dropdownItem -> {
+			dropdownItem.setHref(
+				_renderResponse.createRenderURL(), "mvcPath",
+				"/edit_data_provider.jsp", "redirect",
+				PortalUtil.getCurrentURL(request), "groupId",
```
Comment (pedroqueiroz) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java]: use _formAdminRequestHelper.getRequest() instead

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java [hunk 5]
```diff
@@ -400,12 +524,55 @@ else if (ArrayUtil.contains(displayViews, displayStyle)) {
...
 
+	protected String[] getDisplayViews() {
+		return _DISPLAY_VIEWS;
+	}
+
+	protected List<DropdownItem> getFilterNavigationDropdownItems() {
+		HttpServletRequest request = PortalUtil.getHttpServletRequest(
+			_renderRequest);
+
+		return new DropdownItemList(request) {
```
Comment (pedroqueiroz) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java]: use _formAdminRequestHelper.getRequest() instead

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java [hunk 6]
```diff
@@ -400,12 +524,55 @@ else if (ArrayUtil.contains(displayViews, displayStyle)) {
...
+			dropdownItem.setHref(getPortletURL(), "orderByCol", orderByCol);
+			dropdownItem.setLabel(orderByCol);
+		};
+	}
+
+	protected DropdownItemList getOrderByDropdownItemList() {
+		HttpServletRequest request = PortalUtil.getHttpServletRequest(
+			_renderRequest);
+
+		return new DropdownItemList(request) {
```
Comment (pedroqueiroz) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/context/DDMDataProviderDisplayContext.java]: use _formAdminRequestHelper.getRequest() instead

**Comment by pedroqueiroz** [comment]
@vicnate5, this one also needs fixed. :/

**Comment by pedroqueiroz** [comment]
ci:test

**Comment by pedroqueiroz** [comment]
ci:stop

**Comment by pedroqueiroz** [comment]
ci:stop

**Comment by vicnate5** [comment]
ci:test:relevant

**Pull request closed without merging**